readme.md

```diff
@@ -84,6 +84,7 @@ Feel free to play around with the code and fork this repl at [https://repl.it/@v
 - [gomoku-terminal](https://github.com/acrazing/gomoku-terminal) - Play online Gomoku in the terminal.
 - [cfpush](https://github.com/mamachanko/cfpush) - An interactive Cloud Foundry tutorial in your terminal.
 - [startd](https://github.com/mgrip/startd) - Turn your React component into a web app from the command-line.
+- [wiki-cli](https://github.com/hexrcs/wiki-cli) - Search Wikipedia and read summaries directly in your terminal.
```

Comment:
I think you should spend some more time on your readme there. The point of this list is to show off Ink usage, and you don't even have a screenshot (or better yet, animated GIF) of how it looks.

Reply:
Will do! Thanks for the tips.